src/Symfony/Component/Security/Http/composer.json

```diff
@@ -18,14 +18,15 @@
     "require": {
         "php": ">=7.2.5",
         "symfony/deprecation-contracts": "^2.1",
-        "symfony/security-core": "^5.2",
+        "symfony/security-core": "^5.3",
         "symfony/http-foundation": "^5.2",
         "symfony/http-kernel": "^5.2",
         "symfony/polyfill-php80": "^1.15",
         "symfony/property-access": "^4.4|^5.0"
     },
     "require-dev": {
         "symfony/cache": "^4.4|^5.0",
+        "symfony/password-hasher": "^5.3",
```

Comment:
not needed

Reply:
needed it is, password-hasher interfaces are used in tests

Reply:
but it comes as a mandatory transitive dep, isn't it?

Reply:
indeed, sorry! removed